src/hooks/sdk/docs/useMediaStreamMetrics.stories.mdx

```diff
@@ -30,28 +40,35 @@ The hook depends on the `AudioVideoProvider` being rendered. If you are using th
 
 The `videoStreamMetrics` in return value is a map of attendee to stream of video metrics mappings. The format of `videoStreamMetrics` is shown below:
 
-```
-videoStreamMetrics {
-  attendeeId:{ // remote attendees will have downstream metrics
+```ts
+videoStreamMetrics: {
+  remoteAttendeeId: {
```

Comment:
If we are changing the key, this would be a breaking change for builders to handler the new keys right?

Reply:
Ahh, this is just documentation and `remoteAttendeeId` and `localAttendeeId` will be actual `attendeeId`s right?

Reply:
No I didn't actually change the key, just make it a bit more clear. The key is `attendeeId`, the uuid.